app/javascript/responseTemplates/responseTemplates.js

```diff
@@ -0,0 +1,225 @@
+/* eslint-disable no-alert */
+/* eslint-disable no-restricted-globals */
+function buildModResponseHTML(response) {
+  const array = response
```

Comment:
consider a more descriptive name for the variable instead of the generic `array`.